### PR TITLE
add ssl connection pool

### PR DESCRIPTION
**Pull request by HarrisChu**
as title

### CODE REVIEW

File: example/nebula-test-ssl.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+// initial session for every vu
+var session = pool.getSession("root", "nebula")
+session.execute("USE ldbc")
+// export let options = {
+// 	stages: [
+// 		{ duration: '2s', target: 20 },
+// 		{ duration: '2m', target: 20 },
+// 		{ duration: '1m', target: 0 },
+// 	],
+// };
```
Comment (darionyaphet) [inline example/nebula-test-ssl.js:20]: if this PR is ready please clean the comment thanks

File: client.go [hunk 2]
```diff
@@ -10,31 +11,78 @@ import (
...
+
+	output struct {
+		timeStamp    int64
+		nGQL         string
+		latency      int64
+		responseTime int32
+		isSucceed    bool
+		rows         int32
+		errorMsg     string
+	}
```
Comment (Nicole00) [inline client.go:63]: no need errorCode?

File: client.go [hunk 3]
```diff
@@ -83,57 +101,105 @@ func New() *NebulaPool {
 	}
 }
 
+// NewSSLConfig return sslConfig
+func (np *NebulaPool) NewSSLConfig(rootCAPath, certPath, privateKeyPath string) {
+	np.sslconfig = &sslConfig{
+		rootCAPath:     rootCAPath,
+		certPath:       certPath,
+		privateKeyPath: privateKeyPath,
+	}
```
Comment (Nicole00) [inline client.go:110]: For SelfSign, is password in privateKeyPath file?

**Review by Nicole00 — APPROVED** [review]

File: client.go (hunk 2 above)
Comment (HarrisChu) [reply client.go:63]: error code is less useful for user.

just like this
image

File: client.go (hunk 3 above)
Comment (HarrisChu) [reply client.go:110]: nebula-go doesn't support password yet ..

**Pull request merged**